src/components/media/mediaFigCaption.tsx

```diff
@@ -0,0 +1,43 @@
+// ----- Imports ----- //
```

Comment:
nitpick, but should it be called `mediaFigcaption`? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption

Reply:
i guess i was following the pattern in the code base of `figCaption`